### ISSUE TITLE
after 8.0.0 not possible to use in CJS project

### ISSUE THREAD

**Issue by swarnadeepsaha** [gajus/lightship#64]
after `8.0.0` is not able to use in the CJS project reason it has no export definition for CJS `require`.
not able to move beyond `7`.

**Issue: Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in /usr/app/bin/node_modules/lightship/package.json**

Please add a build for CJS too and redefine the export similarly to
```
"exports": {
    "require": "./cjs/index.js",
    "import": "./esm/index.js"
  },
```

**Comment by dsstewa** [comment]
Also having this issue

**Comment by throrin19** [comment]
same problem here

**Comment by gajus** [comment]
fixed in v9.0.1

**Comment by throrin19** [comment]
Same problem : 

```
Error [ERR_REQUIRE_ESM]: require() of ES Module xxx/nodejs/kube-collector/node_modules/lightship/dist/cjs/index.js from xxx/xxx/xxx.js not supported.
Instead change the require of index.js in xxx/xxx/xxx.js to a dynamic import() which is available in all CommonJS modules.
```
